rocketpy/AeroSurfaces.py

```diff
@@ -329,7 +352,7 @@ def allInfo(self):
         return None
 
 
-class Fins(ABC):
+class Fins(AeroSurfaces, ABC):
```

Comment:
I might be wrong, but, since `AeroSurfaces` is an Abstract Class, its child already inherits `ABC`.

```suggestion
class Fins(AeroSurfaces):
```